packages/sharec-schema/lib/actions/compose.js

```diff
@@ -27,13 +27,17 @@ const applySchemaByKeys = ({ schema, keys = [], target }) =>
     for (const key of keys) {
       const strategy = schema[key] || schema.$$default
       const ignoreList = schema.$$ignore || []
+      const current = params.current.get(key)
 
       if (!strategy || !params.upcoming.has(key) || ignoreList.includes(key)) {
+        if (current !== undefined) {
```

Comment:
I guess, better to set `current` to target before this condition. Probably, It should be processed correctly.
Main point – to reduce nested conditions, i.e – there is shouldn't be `if` inside `if`.

Reply:
OK, I've removed the nested if and removed the old `target.set(key, params.current.get(key))` (it shouldn't have been there)